README.md

```diff
@@ -75,25 +75,24 @@ public void Configure(IApplicationBuilder app)
     // this is required for websockets support
     app.UseWebSockets();
 
-    // use websocket middleware for ChatSchema at path /graphql
-    app.UseGraphQLWebSockets<ChatSchema>("/graphql");
+    // use websocket middleware for ChatSchema at default url /graphql
```

Comment:
Internally our team tends to stick with "path" for the url path and "url" for an absolute one, but that's just nitpicking in a doc file. You could make it "url path" if you wanted but eh, up to you.

Reply:
OK